server/pkg/nostr/client.go

```diff
@@ -110,16 +110,6 @@ func (c *Client) GenerateKeyPair(ctx context.Context) (string, error) {
 	return nostr.GenerateKeyPair(), nil
 }
 
-// ConnectToRelay connects to a relay with a given url
-func (c *Client) ConnectToRelay(ctx context.Context, conState jsonrpc.State, url string) error {
-	state := State(conState)
-	if state.Client == nil {
-		return pkg.ErrClientNotConnected{}
-	}
-
-	return state.Client.ConnectAuthRelay(ctx, url)
-}
-
```

Comment:
What's the reason this function is removed?

Reply:
yes, this is a duplicate:
https://github.com/threefoldtech/web3_proxy/blob/910fc4fb2dd0b7f453ca78c6b6d1de061f5c569d/server/pkg/nostr/client.go#L89-L96